src/components/plots/abstractPlot.ts

```diff
@@ -149,12 +149,13 @@ export module Plot {
      *
      * Here's a common use case:
      * ```typescript
-     * plot.attr("r", function(d) { return d.foo; });
+     * plot.attr("x", function(d) { return d.foo; }, xScale);
      * ```
-     * This will set the radius of each datum `d` to be `d.foo`.
+     * This will set the x accessor of each datum `d` to be `d.foo`,
+     * scaled in accordance with `xScale`
      *
      * @param {string} attrToSet The attribute to set across each data
-     * point. Popular examples include "x", "y", "r". Scales that inherit from
+     * point. Popular examples include "x", "y". Scales that inherit from
      * Plot define their meaning.
```

Comment:
"Scales that inherit from Plot define their meaning"

... This doesn't make any sense.